src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -200,6 +200,7 @@ internal CommandLineParameterParser(PSHostUserInterface hostUI, string bannerTex
             _helpText = helpText;
         }
 
+#region Internal properties
         internal bool AbortStartup
```

Comment:
Should `#region` be in position with the line?

Reply:
updated